src/SharedClasses/ChatMessage.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace SharedClasses;
+
+public class ChatMessage
```

Comment:
Consider object validity and encapsulation.

Reply:
As per our conversation the parameterless constructor is required in order to deserialize the class.
I've added a note in the code to reflect this